aurora-answers.rb

```diff
@@ -0,0 +1,58 @@
+# problem 1
+
+def mirror(list)
+
+  # returns a new array
+  # doubles size of orig array
+  # appends mirror image of original array on orig
+
+  result = []
+
+  list.length.times do |i|
+    result << list[i]
+  end
+
+  rev = -1
+  list.length.times do
+    result << list[rev]
+    rev -= 1
+  end
```

Comment:
This works and is easy to follow, but if requires iterating through the list twice. Can you think of a way to do this only iterating through the list once? (check the solutions branch for my solution)